### PR TITLE
Avangelista.CowabungaLite 1.0.1

### PR DESCRIPTION
**Pull request by Macleykun**
- [x ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ x] This PR only modifies one (1) manifest
- [ x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---
same as 1.0 but update hash and url and version (same adjust for itunes is applied)
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/120899)

### CODE REVIEW

**Comment by Macleykun** [comment]
I'm going to assume that the manifest is valid tho?

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installation failed with exit code -1978335141 

image

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 2023-09-27 16:44:52.108 [CLI ] Archive malware scan passed  
> 2023-09-27 16:44:52.109 [CLI ] Extracting archive to: C:\Users\User\AppData\Local\Temp\WinGet\Avangelista.CowabungaLite.1.0.1\extracted  
> 2023-09-27 16:45:06.800 [CLI ] Successfully extracted archive  
> 2023-09-27 16:45:06.802 [CLI ] Unable to locate nested installer at: C:\Users\User\AppData\Local\Temp\WinGet\Avangelista.CowabungaLite.1.0.1\extracted\./CowabungaLite/CowabungaLite.exe  
> 2023-09-27 16:45:06.803 [CLI ] Terminating context: 0x8a15005b at D:\a\_work\1\s\external\pkg\src\AppInstallerCLICore\Workflows\ArchiveFlow.cpp:62

File: manifests/a/Avangelista/CowabungaLite/1.0.1/Avangelista.CowabungaLite.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+PackageIdentifier: Avangelista.CowabungaLite
+PackageVersion: "1.0.1"
+InstallerLocale: en-US
+InstallerType: zip
+ReleaseDate: 2023-09-26
+Installers:
+- Architecture: x64
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: './CowabungaLite/CowabungaLite.exe'
```
Comment (stephengillie) [inline manifests/a/Avangelista/CowabungaLite/1.0.1/Avangelista.CowabungaLite.installer.yaml:13]:
```suggestion
  - RelativeFilePath: '\CowabungaLite-1.0.1\CowabungaLite.exe'
```

**Comment by Macleykun** [comment]
lmao we finnaly got it XD

**Review by stephengillie — APPROVED** [review]

**Pull request merged**